code/modules/power/supermatter/supermatter.dm

```diff
@@ -438,10 +435,7 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_shard)
 					L.rad_act(rads)
 
 			explode()
-
-
-	power -= ((power/500)**3) * powerloss_inhibitor
-
+			
 	return 1
 
 /obj/machinery/power/supermatter_shard/bullet_act(obj/item/projectile/Proj)
```

Comment:
there's another spaceturf check in bullet_act. Is it worth having a look at that?